academy/5-cosmjs/multi-msg.md

```diff
@@ -259,7 +259,14 @@ export interface MsgDelegate {
 }
 ```
 
-Now that you know the `typeUrl` for delegating some tokens is `/cosmos.staking.v1beta1.MsgDelegate`, you need to find a validator's address that Alice can delegate to. Find a list of validators in the [testnet explorer](https://explorer.theta-testnet.polypore.xyz/validators). Select a validator and use their address in the following script, which you can copy to replace your original token transfer:
+Now that you know the `typeUrl` for delegating some tokens is `/cosmos.staking.v1beta1.MsgDelegate`, you need to find a validator's address that Alice can delegate to. Find a list of validators in the [testnet explorer](https://explorer.theta-testnet.polypore.xyz/validators). Select a validator and set their address as a variable:
+
+```
```

Comment:
Can you add `typescript` after the 3 back ticks? So we get the correct code highlighting.

Reply:
added